apollo-router-core/src/query_planner/router_bridge_query_planner.rs

```diff
@@ -23,7 +23,12 @@ impl RouterBridgeQueryPlanner {
 
 #[async_trait]
 impl QueryPlanner for RouterBridgeQueryPlanner {
-    #[tracing::instrument(name = "plan", level = "debug")]
+    #[tracing::instrument(
+        skip_all,
+        fields(query = %query),
```

Comment:
I think there was a decision a few weeks ago about not sending the query as part of the traces if we do not have a good way to sanitize them (ex: remove variables with potentially private information). I cannot find the original discussion but there's this issue: https://github.com/apollographql/router/issues/48

Reply:
Oh that makes sense, lemme remove that